tests/src/common/WskCli.java

```diff
@@ -968,11 +982,9 @@ public RunResult cli(boolean verbose, int expectedExitCode, String... params) th
      * @return RunResult which contains stdout,sterr, exit code
      */
     public RunResult cli(boolean verbose, int expectedExitCode, File workingDir, String... params) throws IllegalArgumentException, IOException {
-        String[] cmd = WhiskProperties.useCliDownload() ? new String[] { getDownloadedCliPath() } : new String[] { WhiskProperties.python, new File(cliDir, binaryName).toString() };
-        String[] args = verbose ? Util.concat(cmd, "--verbose") : cmd;
-        if (binaryName.equals("wsk")) {
-            args = Util.concat(cmd, "--apihost", WhiskProperties.getEdgeHost());
-        }
+        String[] cmd = WhiskProperties.useCliDownload() ? new String[] { getDownloadedCliPath() } : new String[] { new File(cliPath).toString() };
+        cmd = verbose ? Util.concat(cmd, "--verbose") : cmd;
+        String[] args = Util.concat(cmd, "--apihost", WhiskProperties.getEdgeHost());
```

Comment:
here: you can add `-i` once (and later you can make this sensitive to the api host value.